cmake.Modules/CompileFlags.cmake

```diff
@@ -24,7 +24,7 @@ IF(UNIX)
         SET(CMAKE_MACOSX_RPATH 1)
     ENDIF()
 
-    SET (CXX_OPTIONS       " -std=c++11 -Wno-deprecated -Wall -Werror -Wshadow -Wextra -Wfloat-equal -Wpointer-arith -Wwrite-strings -Wpacked")
+    SET (CXX_OPTIONS       " -std=c++11 -Wno-deprecated -Wall -Wextra -Wfloat-equal -Wpointer-arith -Wwrite-strings -Wpacked")

```

Comment:
Removing -Werror is violating our coding guidelines. What is the motivation for that?